modules/database/migrations/sqls/20190507194131-add-payment-profiles-up.sql

```diff
@@ -0,0 +1,16 @@
+
+/* Simple payment profile with min collateral and collateral target */
+create table payment_profiles (
+	id bigserial primary key,
+  minimum_maintained_collateral_wei wei_amount default '0',
```

Comment:
Minor nitpick, if you're going to default it like this, you might as well make it `not null`.